### ISSUE TITLE
Data Dictionary

### ISSUE THREAD

**Issue by dwalt** [adiwg/mdTranslator#22]
I noticed looking through the Schema Viewer that under domain>member we have three elements: name, value and definition. I'm not sure what is meant by name and value. But the elements needed are code and name. Code being the coded value in the data resource, and name the English value associated with the code.

**Comment by stansmith907** [comment]
Value is code.  I think we strayed from codes since it may be a numeric
value, alpha code, or word or phrase.



*From:* dwalt [mailto:notifications@github.com]
*Sent:* Tuesday, March 17, 2015 4:46 PM
*To:* adiwg/mdTranslator
*Subject:* [mdTranslator] Data Dictionary (#22)



I noticed looking through the Schema Viewer that under domain>member we
have three elements: name, value and definition. I'm not sure what is meant
by name and value. But the elements needed are code and name. Code being
the coded value in the data resource, and name the English value associated
with the code.

—
Reply to this email directly or view it on GitHub
<https://github.com/adiwg/mdTranslator/issues/22>.
